app/views/attendances/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= form_for(@attendance) do |f| %>
+  <% if @attendance.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@attendance.errors.count, "error") %> prohibited this attendance from being saved:</h2>
+
+      <ul>
+      <% @attendance.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :date %><br>
+    <%= f.text_field :date %>
+    <%= f.label :present %><br>
+    <!-- #Don't know what to put here -->
```

Comment:
Here I'm just talking about the view. In your controller, one of the params you're allowing is `:student_id`. That means we're expecting some kind of parameter named `student_id` to be in the params. However, in this form, there's no field called `student_id`.

Since `student_id` represents a student, it makes a lot of sense to me to have a dropdown of all students for the user to select from. I'm suggesting that you could use very similar code from the CourseLoads form to do this.